test/views/website/CaptureWebsiteViewSpec.scala

```diff
@@ -70,10 +70,6 @@ class CaptureWebsiteViewSpec extends ViewBaseSpec {
             elementText(Selectors.pageHeading) shouldBe "What’s the website address?"
           }
 
-          "have the correct hint text" in {
```

Comment:
Same comment as above. If we are missing these hint tests for mobile and landline it would be worth adding them.